python/pyhail/dataset.py

```diff
@@ -689,7 +689,71 @@ def linreg(self, y, covariates="", root="va.linreg", minac=1, minaf=0.0):
         pargs = ['linreg', '-y', y, '-c', covariates, '-r', root, '--mac', str(minac), '--maf', str(minaf)]
         return self.hc.run_command(self, pargs)
 
-    def logreg(self, test, y, covariates=None, root=None):
+    def lmmreg(self, y, kinshipfilter, covariates="", assocfilter=None, noassoc=False,
+               useml=False, globalroot=None, varoot=None, delta=None,
+               sparsitythreshold=1.0, forceblock=False, forcegrammian=False):
```

Comment:
If the lmm command takes the kinship vds by name, the kinship `VariantDataset` should be given as an argument like `join`.